samples/out-of-the-box-configurator/README.md

```diff
@@ -1,26 +1,39 @@
 [![Symphony Software Foundation - Incubating](https://cdn.rawgit.com/symphonyoss/contrib-toolbox/master/images/ssf-badge-incubating.svg)](https://symphonyoss.atlassian.net/wiki/display/FM/Incubating) [![Build Status](https://travis-ci.org/symphonyoss/App-Integrations-FE-Commons.svg?branch=dev)](https://travis-ci.org/symphonyoss/App-Integrations-FE-Commons) [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
-# Integrations Front-End Commons Documentation
-
-This document provides a brief overview of Integration Front-End Commons components and how to build and publish them.
-
-# Overview
-
-Integration FE Commons provides the common components required to build a configurator application for Symphony Platform integrations, such as room search, webhook creation, webhook editing etc.
-
-# Installation instructions for the Javasctipt developer
-
-### What you’ll build
-You’ll build a simple Javascript library that provides some useful services to build new configurator apps for integrations.
-
-### What you’ll need
-* Nodejs 6.10
-* NPM node-license-validator
-* NPM semantic-release
-
-### Build with NPM
-To start from scratch, do the following:
-
-1. Clone the source repository using Git: `git clone git@github.com:symphonyoss/App-Integrations-FE-Commons.git`
-2. cd into _App-Integrations-FE-Commons_
-3. Build using npm: `npm install`
+# Out-of-the-Box Sample
+This document provides information about how to use this sample integration.
+
+## About this sample
+This sample shows the simplest way to create a new configurator app, by using all the out-of-the-box behavior implemented by FE-Commons.
+
+## What you’ll build
+You'll build an integration sample where you can create instances that notifies you whenever a webhook is triggered.
+
+## Requirements
+In order to run the configurator sample you'll need [Node 6.10.x][node] and [npm][npm] installed.
+
+## Instructions for Javascript Developers 
+1. Clone or download the App-Integrations-FE-Commons repository
+2. In the out-of-the-box sample folder run `npm install`
+3. After installing the dependencies run `npm run watch`
+
+The samples folder is set up with multiple development samples built up with [webpack][webpack] and [eslint][eslint]. Every time you change a file and save it, eslint checks your code sintaxe and webpack reloads your browser. This sample is buit up with React and some imported components from this repo.
```

Comment:
I think webpack does not reload the browser itself, it only recompiles the bundle file in the served folder